content/docs/gke/authentication.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Authentication and authorization to GCP"
+weight = 4
++++
+
+
+# Authenticating gcloud
```

Comment:
Should we clarify that this section is for authenticating from outside the Kubeflow cluster?

gcloud & kubectl can also be run inside the cluster (e.g. from a Jupyter notebook)